src/main/resources/templates/friends.html

```diff
@@ -0,0 +1,27 @@
+{# The page to display user's friends #}
+{% extends "base/li-sidebar-page" %}
+{% block title %} Exception Null - Buddy List {% endblock %}
+
+{% macro friendCard(imageSrc, username, userType, online, rating) %}
+<div class="card bottom-aligned">
+    <div class="profile-container">
+        <img class="profile-photo" src={{ imageSrc }} />
+    </div>
+    <div class="username">{{ username }}</div>
+    <div class="user-type">{{ userType }} - <span class="rating">★ {{ rating }}</span></div>
+    <div class={{ online ? "status-online" : "status-offline" }}>{{ online ? "Online" : "Offline" }}</div>
```

Comment:
The padding on the card seems to ignore this element
![example](http://i.imgur.com/v3g0H8u.png)